src/parser/expr.js

```diff
@@ -272,7 +272,7 @@ module.exports = {
         return this.node('cast')('int', this.next().read_expr());
 
       case this.tok.T_DOUBLE_CAST:
-        return this.node('cast')('double', this.next().read_expr());
+        return this.node('cast')('float', this.next().read_expr());
```

Comment:
not sure in PHP but in principle double and float have not the same precision, I'll check this

Reply:
:smile: got it from https://github.com/glayzzle/php-parser/blob/master/src/lexer.js#L111, actually there is only one token to these 3 casts types

Reply:
Exactly - they are just aliases in PHP :wink: